packages/oui-tile/tile.less

```diff
@@ -89,6 +89,10 @@
     margin-left: 0;
     word-break: break-all;
 
+    & .oui-paragraph {
```

Comment:
This should be set directly on `.oui-paragraph` style. No need to add that in `.oui-tile` style.

Reply:
Done